CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## UNRELEASED
```

Comment:
I'm afraid I need to inform you that vodozemac never abandoned `git-cliff` for changelog entries. So this belongs in a commit message.

Reply:
ah, right. Presumably putting it in the subject line of the PR, and then squash-merging the PR, will do the right sort of thing?

Reply:
As long as it's a conventional commit it will be picked up, since you historically complained that commit messages don't make great changelog entries you can put the conventional commit into a merge commit.

The merge commit message is anyways useless so it's a good opportunity to repurpose it for a better cause.

In the future I'd like to make it possible so you can have a special tag in merge commits which will mark the start of the changelog entry, similar to how the `Signoff` is treated specially in commit messages.